api/CaretPosition.json

```diff
@@ -0,0 +1,52 @@
+{
+  "api": {
+    "CaretPosition": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/CaretPosition",
+        "support": {
+          "webview_android": {
+            "version_added": false
+          },
+          "chrome": {
+            "version_added": false
+          },
+          "chrome_android": {
+            "version_added": null
```

Comment:
@jpmedley I would assume that Chrome Android doesn't support it, if Chrome doesn't. Do you have any tips how to find out about exceptions where support is not consistent between Chrome variants?

Reply:
I think they are generally the same. I updated this.